lib/modules/pageNavigator.js

```diff
@@ -39,20 +45,32 @@ module.afterLoad = () => {
 	if (module.options.toTop.value) {
 		backToTop();
 	}
+	if (module.options.toComment.value) {
```

Comment:
Can this also test if the comment area is available? In case user is logged out or banned. (Probably should be a utility function, although dunno which section that should go in.)